pyshtools/shio/common.py

```diff
@@ -0,0 +1,28 @@
+"""
+Common routines for I/O
+"""
+
+import numpy as _np
+
+def _derivative(epoch, ref_epoch, trnd, periodic):
```

Comment:
Does this routine calculate a "derivative"? Not sure if I understand the naming...